src/CheckBox/readme.md

```diff
@@ -23,28 +23,36 @@ This React component is based on SEB Bootstrap style. Supports customization and
 
 ```html
 <CheckBox
-    name="checkbox"
     label="Checkbox label"
-    checked={this.state.checkbox}
-    onChange={(event) => { this.setState({ checkbox: event.target.checked }); }}
+    checked={checkboxValue}
+    onChange={onChangeHandler}
 />
 ```
 
 ## Properties
 
-These are the current available properties:
-
-| Property     | Type                                                   | Description                                                            |
-| ------------ | ------------------------------------------------------ | ---------------------------------------------------------------------- |
-| checked      | `boolean`                                              | The checked value                                                      |
-| className?   | `string`                                               | Custom class                                                           |
-| description? | `string`                                               | A option description                                                   |
-| disabled?    | `boolean`                                              | Disabled status                                                        |
-| id?          | `string`                                               | Id property                                                            |
-| inline?      | `boolean`                                              | Renders inline checkbox                                                |
-| label        | `string`                                               | Checkbox label                                                         |
-| name         | `string`                                               | Name property                                                          |
-| onChange     | `(event: React.ChangeEvent<HTMLInputElement>) => void` | On change action                                                       |
-| reference?   | `React.RefObject<HTMLInputElement>`                    | React Ref obj                                                          |
-| topLabel?    | `string`                                               | A label which shows on top of component                                |
-| condensed?   | `boolean`                                              | Condenses the checkboxes. Usefull when rendering them below each other |
+This component extends all native attributes of `HTMLInputElement`, while offering the following customizations:
+
+| Property     | Type                    | Description                                                 |
+| ------------ | ----------------------- | ----------------------------------------------------------- |
+| description? | `string`                | Description to be displayed underneath the checkbox element |
```

Comment:
it's underneath? I thought it used to be next to it.

Reply:
not anymore, it messes up the UI especially when you have a long description. Displaying the description on its own line not just looks better, it behaves better.